src/renderer/prefComponents/imageUploader/index.vue

```diff
@@ -1,19 +1,28 @@
 <template>
   <div class="pref-image-uploader">
     <h4>Image Uploader</h4>
-    <section class="current-uploader">
-      <div>The current image uploader is <span class="uploader">{{currentUploader}}</span>.</div>
-    </section>
-    <separator></separator>
     <section class="configration">
       <el-tabs v-model="activeTab">
+        <el-tab-pane label="Overview" name="none">
+          <div v-if="currentUploader !== 'none'">
+            <div>The current image uploader is {{ getServiceNameById(currentUploader) }}.</div>
+
+            <!-- TODO: Reset all uploader settings and secret values. -->
+            <el-button class="btn-reset" size="mini" disabled>Reset</el-button>
```

Comment:
What's the button used for? it's disabled.

Reply:
This button should be used to remove all image uploader settings and also empty out the secret values but I removed it for know. It was disabled because I don't wanted to implement this but reverse it for later.